ci/docker/Dockerfile.maldet.leap.15

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018-2022 Intel Corporation
+# All rights reserved.
+#
+# 'recipe' for Docker to build an environment scan DAOS for malware.
+#
+
+# Pull base image
+ARG BASE_DISTRO=opensuse/leap:15.3
```

Comment:
15.3?  Aren't we on 15.4 now?

Reply:
Unfortunately no.   Moving to 15.4 is currently blocked by a few issues.
The docker build command to the dockerfile now controls which 15.x version is used, so that we can move to a newer 15.x release with probably no edits to the Dockerfile, and for GitHub Actions, it can be configured to use the same Dockerfile to build for multiple point releases.

Reply:
The daos build and all GitHub actions are all 15.4 aren't they?  What are the issues you refer to and do they only apply to maldet?

Reply:
I do not know about GitHub actions, but all CI builds and tests are at Leap 15.3.
As I understand it there are two issues;

- Mock builds of leap 15.4 are currently broken due to an OpenSUSE bug.
- DAOS built for Leap 15.3 requires some Leap 15.3 repos to be enabled on Leap 15.4 because of a library change.

Has nothing to do with Maldet or C;lamAV.

Reply:
Ah, I see.  The Leap/icc build in Jenkins is 15.4 as are all GitHub actions builds but none of them build packages.